src/autopas/containers/ParticleContainerInterface.h

```diff
@@ -172,10 +172,22 @@ class ParticleContainerInterface {
   virtual void deleteAllParticles() = 0;
 
   /**
-   * Get the total number of particles saved in the container (owned and halo but not dummies).
-   * @return Number of particles in the container.
+   * Get the number of particles with respect to the specified IteratorBehavior.
+   * Note: Since this function counts the number of the respective particles in the internal particle storage, this is
```

Comment:
```suggestion
   * @warning Since this function counts the number of the respective particles in the internal particle storage, this is
```

Reply:
done in [37e2621](https://github.com/AutoPas/AutoPas/pull/782/commits/37e26218410380f7bc586d66bd83b316d23a3d28)